src/vos/evtree.c

```diff
@@ -2511,8 +2535,16 @@ evt_common_insert(struct evt_context *tcx, struct evt_node *nd,
 		ne->ne_child = in_off;
 	}
 
-	if (!reuse)
+	if (!reuse) {
+		if (!tx_added) {
+			rc = umem_tx_add_ptr(evt_umm(tcx), &nd->tn_nr,
+					     sizeof(nd->tn_nr));
+			if (rc < 0)
+				return rc;
+		}
+
 		nd->tn_nr++;
+	}
```

Comment:
I see, so the problem is that when inserting into evtree, some fields of interior node are being modified but missed being added to transaction by PMDK add_range call, is that correct? When I read the commit comment, I was thinking you are going to call add_range for newly allocated range. Could you revise the commit comment a bit (maybe keep the first paragraph only?)? The second part of the comment is bit misleading.